client/src/screens/groups/components/GroupDetailsRenderItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Alert,
+} from 'react-native';
+
+import EventListItem from '../../../components/Events/EventListItem';
+import ScheduleListItem from '../../../components/Schedules/ScheduleListItem';
+import UserListItem from '../../../components/Users/UserListItem';
+
+const GroupDetailsRenderItem = ({ item }) => {
+  // eslint-disable-next-line no-underscore-dangle
+  switch (item.__typename) { // GQL typename from Schema
+    case 'User':
+      return (
+        <UserListItem
+          user={item}
+      // TODO: Deep link
+          onPress={() => Alert.alert('PLACEHOLDER', 'Deeplink to user details')}
```

Comment:
when you do get around to implementing this I think it'd be best to do it as a callback that provides the type and the id so the caller can decide how to route it.

Reply:
Agreed. everything is there for the link to work just need the plumbing